code/game/machinery/Sleeper.dm

```diff
@@ -19,12 +19,12 @@
 	var/list/available_chems
 	var/controls_inside = FALSE
 	var/list/possible_chems = list(
-		list("epinephrine", "morphine", "salbutamol", "bicaridine", "kelotane"),
-		list("oculine","inacusiate"),
-		list("antitoxin", "mutadone", "mannitol", "pen_acid"),
+		list("epinephrine", "morphine", "salbutamol", "ether"),

```

Comment:
Remove morphine, add silver sulf and styptic powder

Reply:
Styptic powder does toxin damage when ingested, we want to add this to sleepers?

Reply:
Added